index.html

```diff
@@ -347,9 +151,70 @@ <h3>Skills</h3>
       <img id="indicator" src="assests/images/shape.png" alt="Home Indicator/Light">
     </div>
   </footer>
-
+  
+    <!-- Popup Window -->
+    <div class="modal-overlay">
+      <div class="modal-container">
+          <div class="modal-header">
+            <div class="heading">
+              <span class="close-btn">&times;</span>
```

Comment:
- [ ] Kindly reduce the size of the close button for both mobile and desktop so that it matches the chosen Figma design.